atlasdb-processors-tests/src/test/java/com/palantir/processors/AutoDelegateInterfaceTests.java

```diff
@@ -96,4 +107,10 @@ public void generatedInterfaceCallsMethodOnDelegate() {
         instanceOfInterface.methodWithReturnType();
         verify(mockImpl, times(1)).methodWithReturnType();
     }
+
+    private Set<String> extractNonStaticMethods(Class klass) {
```

Comment:
nit: `static`

Reply:
(This method was moved to `TestingUtils`, and is static there.)